swap/swap_test.go

```diff
@@ -1242,8 +1260,8 @@ func TestPeerGetLastSentCumulativePayout(t *testing.T) {
 	_, peer, clean := newTestSwapAndPeer(t, ownerKey)
 	defer clean()
 
-	if peer.getLastSentCumulativePayout() != 0 {
-		t.Fatalf("last cumulative payout should be 0 in the beginning, was %d", peer.getLastSentCumulativePayout())
+	if !peer.getLastSentCumulativePayout().Equals(uint256.FromUint64(0)) {
```

Comment:
Wouldn't it be better to compare just to the `New()` value instead to run `FromUint64`?

Reply:
if you are reading this line by itself, how do you know that:
- `New()` will result in a value which can safely be compared through `Equals`?
- assuming `New()` results in a safe value, that this value is the minimum valid value? (and maybe even: that the minimum valid value is `0`) 

i will concede that this is a verbose mess considering we just want to say `zero`. however: i think it's clearer this way. 

i am happy to discuss it further though

Reply:
I am pointing this out only because I have seen elsewhere you have used just that `New()` for a zero value, for example for the return value for  `getLastSentCumulativePayout`